app/controllers/planets_controller.rb

```diff
@@ -0,0 +1,24 @@
+class PlanetsController < ApplicationController
+  def index
+    @planets = Planet.all
+  end
+
+  def new
+    @planet = Planet.new
+  end
+
+  def create
+    @planet = Planet.new(planets_params)
+    if @planet.save
+      redirect_to planets_path
```

Comment:
Maybe it could redirect_to planet_path ? 
So it goes to the show page and not index